data/fields/parking/orientation.json

```diff
@@ -0,0 +1,12 @@
+{
+    "key": "parking:orientation",
+    "type": "combo",
+    "label": "Parking bay orientation",
+    "strings": {
+        "options": {
+            "parallel": "Parallel to the carriage way",
```

Comment:
Don't have a strong opinion on this, but I would recommend simplifying this part as well too.
(Parallel, diagonal, perpendicular).
Or we could do short versions: Parallel to street, etc.. (using 'street' here as it is more inline with the 'Street-side' parking)

Reply:
Yes, field options should be as brief as possible, because the inspector sidebar can be resized to a rather narrow width and some translations may require more characters than in English.

For consistency with other field options, use Title Case instead of Sentence case. Sentence case is suitable for tooltips, which to my knowledge aren’t supported by schema-builder yet.

Reply:
I think I'll stick with the current descriptions, as these match with the proposal and wiki documentation.

Reply:
The wiki documentation isn’t intended to dictate the precise UI for an editor field; we do have some leeway to tighten up the descriptions for a UI as long as it doesn’t contradict the documentation and actual usage. At the very least, the American English locale should use correct English spelling such as “[streetside](https://en.wiktionary.org/wiki/streetside)” and “[carriageway](https://en.wiktionary.org/wiki/carriageway#English)”. (But note that the term “carriageway” sounds incredibly archaic to a lay American English speaker.)

Moreover, the wiki doesn’t have a problem with other presets and fields using title case.